app/views/foods/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="index-page">
+<div class="add-food">
+<a href="/foods/new" >Add Food</a>
+</div>
+<div class="food-container">
+<div class="food-table">
+
+ <table>
+  <tr>
+    <th>Food</th>
+    <th>Measurement Unit</th>
+    <th>Unit Price</th>
+    <th>Actions</th>
+  </tr>
+  <% @foods.each do |food|%>
+   <tr class="food-row" data-food-id="<%= food.id %>">
+   <td><%=food.name%></td>
+   <td><%=food.measurement_unit%></td>
+   <td>$<%=food.price%></td>
+   <td><%= render partial: 'shared/delete_button', locals: { item: food, delete_url: food_path(food.id) } %></td>
+  </tr>
```

Comment:
[REQUIRED] Please ensure that your design aligns with the wireframe provided. Please take note of the suggestions outlined in the image below.

![image](https://github.com/Lul-Abdifan/Rails_Capstone/assets/62938735/502c602d-bba3-4798-8538-3eec28f6c026)